web/components/TextBlock/Programs/Programs.tsx

```diff
@@ -122,32 +122,45 @@ export const Programs = ({
   const collection = getCollectionForSelectionType(type, allPrograms, programs);
   const venues = collection?.map((program) => program.venues).flat();
   const router = useRouter();
-  const venue = router.query.venue as string;
-  const activeProgram = findByVenueSlug(collection, venue) || programs?.[0];
+  const venueName = router.query.venue as string;
+  const activeProgram = findByVenueSlug(collection, venueName) || programs?.[0];
   if (!activeProgram) {
-    console.error(`No activeProgram found for Venue slug '${venue}'`);
+    console.error(`No activeProgram found for Venue slug '${venueName}'`);
     return null;
   }
 
   if (type === 'all' || type === 'highlighted') {
+    const activeVenue = activeProgram.venues?.[0];
+    const { address } = activeVenue || {};
     const sessionsPerDay = partition(activeProgram.sessions, (session) => {
       const sessions = mapSessionDurationAndIds(activeProgram);
       const id = session.session?._id ?? session._key;
       const date = sessionStart(activeProgram.startDateTime, id, sessions);
-      const timezone = activeProgram.venues?.[0]?.timezone || defaultTimezone;
+      const timezone = activeVenue?.timezone || defaultTimezone;
       return (date && formatDateWithDay(date, timezone, ', ')) || '';
     });
 
     return (
       <>
         <div className={styles.venueNavContainer}>
-          <VenueNav
-            venues={venues}
-            activeVenue={activeProgram.venues[0]}
-            mainVenue={mainVenue}
-          />
+          <VenueNav {...{ activeVenue, venues, mainVenue }} />
         </div>
 
+        {typeof address === 'object' && (
+          <address className={styles.venueAddress}>
+            <div>{address?.name}</div>
```

Comment:
Addresses are actually one case where the [`<br>` element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element) is appropriate, according to the HTML spec.

It would be nice if these address parts could be joined by those instead of being output in separate divs, so we avoid empty divs in the cases where one part is missing. E.g. the missing `name` for Oslo in staging:

```
<div></div><div>Trondheimsveien 2k</div><div>Oslo, 0560</div>
```

vs

```
Trondheimsveien 2k<br>Oslo, 0560
```

Nit if it's hard to do.

Reply:
No problem. 3ae7387